### DISCUSSION TITLE
Courseplay mit Manure System

### DISCUSSION THREAD

**Discussion by GehtKeinenWasAn** [Courseplay/Courseplay_FS22#2999] [Q&A]
Hallo zusammen,
mir ist aufgefallen, dass in Zusammenhang mit dem MOD "Manure System" die Fahrzeuge nicht laden bzw. abladen können. Gibt es dazu schon Infos oder Lösungen???

Vielen Dank schon mal für die Bemühungen...

**Comment by Tensuko** [comment]
Da CP nichts lädt oder ablädt, ist das kein CP Problem.
Wenn Manure System kein AI Unterstützt musss Manure System das einbinden.

**Reply by GehtKeinenWasAn** [reply]
ok Danke...werde dann mal den Modder anschreiben....
einen schönes Wochenende
